src/components/likemark-child/likemark-child.component.js

```diff
@@ -1,29 +1,68 @@
+/* @flow */
+
+// Dependencies
 import React from 'react'
-import PropTypes from 'prop-types'
-import { Card } from 'antd'
 
+// Models
+import type {Likemark as LikemarkType} from 'models/likemark'
+
+// Components
+import { List, Icon, Image } from 'semantic-ui-react'
+import SmoothCollapse from 'react-smooth-collapse'
+import LikemarkDetails from 'components/likemark-details/likemark-details.component'
+
+// Services
+import {utils} from 'services'
+
+// Assets
+import './likemark-child.scss'
+
+// Main component
 /**
  * Display a Likemark Child component.
- * @class @extends React.Component
- * @property {function} props.onClick - Updates the likemark state.
- * @property {name} props.itemType - The name of the children.
- * @fires LikemarkChild#onClick
+ * @property {Function} props.handleLikemarkClick - The function which trigger the state update when a likemark child is selected.
+ * @property {LikemarkType} props.likemarkChild - The likemark child object.
+ * @property {Function} props.toggleMenuVisibility - The function which switch the visibility of the menu component.
+ * @property {boolean} props.expandedDetails - Flag who determines if the details is expanded.
  */
-export default class LikemarkChild extends React.Component {
-  static get propTypes () {
-    return {
-      onClick: PropTypes.func.isRequired,
-      name: PropTypes.string.isRequired
-    }
-  }
-
-  render () {
-    let { onClick, name } = this.props
-
-    return (
-      <li>
-        <Card onClick={onClick}>{name}</Card>
-      </li>
-    )
-  }
+
+type Props = {
+  handleLikemarkClick: (likemarkId: number) => void,
+  likemarkChild: LikemarkType,
+  toggleDetailsVisibilty: () => void,
+  expandedDetails: boolean
+}
+
+export const LikemarkChild = ({handleLikemarkClick, likemarkChild, toggleDetailsVisibilty, expandedDetails}: Props) => {
+  return (
+    <List.Item className='likemark-child'>
+      <Icon
+        name={`angle ${expandedDetails ? 'up' : 'down'}`}
```

Comment:
Ok, I agree and understand, but in my vision is not a so important stuff because we just use this 'up' and 'down' in this file/component.

So, do you still want to me to change this detail?

Reply:
I agree with jcarva on that point

Reply:
i think would be great but it don't really matter. it's up to you.

Reply:
For now let's keep on this way, if we need to reuse in another place, we create/use the enum. And yes, you're right, would be great to have when we need them.